tests/test_cuda_targeted_insertion_mover.py

```diff
@@ -446,12 +446,10 @@ def test_tibd_exchange_deterministic_moves(radius, moves, precision, seed):
         1,
     )
 
-    iterative_moved_coords = conf.copy()
-    for _ in range(moves):
-        iterative_moved_coords, _ = bdem_a.move(iterative_moved_coords, box)
-    batch_moved_coords, _ = bdem_b.move(conf, box)
-    # Moves should be deterministic regardless the number of steps taken per move
-    np.testing.assert_array_equal(iterative_moved_coords, batch_moved_coords)
+    moved_a, _ = bdem_a.move(conf, box)
+    moved_b, _ = bdem_b.move(conf, box)
+    # Moves are only deterministic if the number of steps taken per move is the same
```

Comment:
Think this is actually a bad decision, as it means we can only effectively test the single proposal per move. Going to just switch to constant batch sizes.

As we make more performance modifications need to make sure that we get the same behavior for 1 move vs 1000 moves

Reply:
Thinking about this some more, not sure we can always guarantee that 100 individual moves is going to be identical to 100 moves with incremental. Been trying to avoid repeated partitioning and that changes the ordering of the partition which impacts the selection randomness.

Need to come up with a more robust way to test the behavior in the case of multiple steps

Reply:
Undid in https://github.com/proteneer/timemachine/pull/1225/commits/66241297d52013ea6267787fe6a29eb4588c2397